README.md

```diff
@@ -74,6 +72,16 @@ Valkyrie can be installed as `systemd` service by these steps:
 - unpacking and setting up a configuration file named `config.yml`
 - executing `./svc.sh install` to setup the service
 
+### Swagger
```

Comment:
Moved this down a bit since I felt it didn't flow that well into the surrounding text